gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -32,128 +32,94 @@ export default function IntellCont({ intellContList, publicationType }) {
     return sortedIntellContList
   }
 
-    const sortedIntellContList = sortStatusAndYear(intellContList)
+  const sortedIntellContList = sortStatusAndYear(intellContList)
+  const liInnerHTMLList =
+    sortedIntellContList.filter(element => (element.contype === publicationType || element.contypeother === publicationType)).map(element => {
+      let liString = '';
+      if (element.status) {
+        liString += `${element.status} `
+      }
+      if (element.contype && !element.contypeother) {
+        liString += `${element.contype} - `
+      }
+      if (element.contypeother) {
+        liString += `${element.contypeother} - `
+      }
+      element.intellcont_auth.forEach(elem => {
+        if (elem.lname) {
+          liString += `${elem.lname}, `
+        }
+        if (elem.fname && !elem.mname) {
+          liString += `${elem.fname}; `
+        }
+        if (elem.mname && !elem.fname) {
+          liString += `${elem.mname}.; `
+        }
+        if (elem.fname && elem.mname) {
+          liString += `${elem.fname}. ${elem.mname}.; `
+        }
+      })
+      if (element.status === "Published" && element.dty_pub) {
+        liString += `(${element.dty_pub}). `
+      }
+      if (element.status === "Accepted" && element.dty_acc) {
+        liString += `(${element.dty_acc}). `
+      }
+      if (element.status === "Submitted" && element.dty_sub) {
+        liString += `(${element.dty_sub}). `
+      }
+      if (element.title) {
+        liString += `${element.title}. `
+      }
+      if (element.title_secondary) {
+        liString += `${element.title_secondary} `
+      }
+      if (element.issue && !element.pagenum && !element.volume) {
+        liString += `(${element.issue} ed.). `
+      }
+      if (!element.issue && element.pagenum && !element.volume) {
+        liString += `(pp. ${element.pagenum}). `
+      }
+      if (!element.issue && !element.pagenum && element.volume) {
+        liString += `(vol. ${element.volume}). `
+      }
+      if (!element.issue && element.pagenum && element.volume) {
+        liString += `(vol. ${element.volume}, pp. ${element.pagenum}). `
+      }
+      if (element.issue && !element.pagenum && element.volume) {
+        liString += `(${element.issue} ed., vol. ${element.volume}). `
+      }
+      if (element.issue && element.pagenum && !element.volume) {
+        liString += `(${element.issue} ed., pp. ${element.pagenum}). `
+      }
+      if (element.issue && element.pagenum && element.volume) {
+        liString += `(${element.issue} ed., vol. ${element.volume}, pp. ${element.pagenum}). `
+      }
+      if (element.pubctyst) {
+        liString += `${element.pubctyst}: `
```

Comment:
If the items after this (publisher, web_address) are truly optional -- and based on the if statements, it definitely appears that they are optional -- we would finish off with a colon if we have pubctyst and don't have anything after that. Ending with a colon seems odd.

Reply:
I will add a conditional for each of the following scenarios

1. pubctyst && publisher
2. pubctyst && !publisher
3. !pubctyst && publisher